app/styles/_projects.less

```diff
@@ -43,7 +43,9 @@
       margin-left: 5px;
     }
     a {
+      display: block;
       line-height: initial;
+      .word-break-all();
```

Comment:
I'd almost rather use `overflow: hidden` and truncate the text:

![openshift_web_console_and_developer_tools_-_openshift_web_console_-_https___localhost_9000_](https://cloud.githubusercontent.com/assets/1167259/20064113/8dc9e734-a4d7-11e6-8af6-1ab7b431f144.png)